### PR TITLE
Pk _githubtutorial  : added tutorial

### PR DESCRIPTION
**Pull request by prashantkad**
- added some new contents

### CODE REVIEW

File: pk_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*[ ]  added this file as part of tutorial
+* some more comments
```
Comment (cabmario) [inline pk_tutorial.md:2]: @prashantkad Can you add your username?

**Comment by prashantkad** [comment]
user name is pk

**Comment by prashantkad** [comment]
sorry added to file this time.

**Comment by prashantkad** [comment]
Trying emoji :+1:  :8ball:

**Comment by cabmario** [comment]
Looking good, let's merge

**Comment by prashantkad** [comment]
also fixed bewv thin

**Pull request merged**